app/components/raw-config.js

```diff
@@ -0,0 +1,18 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { later } from '@ember/runloop';
+
+export default Component.extend({
+  copied: false,
+
+  buttonLabel: computed('copied', function () {
+    return this.get('copied') ? 'Copied!' : 'Copy to clipboard';
```

Comment:
Just as a note, `.get` is not mandatory anymore. In most cases you can just `this.copied`. In rare cases when `.get` is required, Ember will tell you so in browser console. No need to fix it here, we still have a lot of `.get`s all over the app, it's just for future reference.